src/algebra/category/Module/simple.lean

```diff
@@ -0,0 +1,49 @@
+/-
+Copyright (c) 2022 Pierre-Alexandre Bazin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pierre-Alexandre Bazin
+-/
+import category_theory.simple
+import algebra.category.Module.abelian
+import ring_theory.simple_module
+
+/-!
+# Simple objects in the category of `R`-modules
+
+We prove simple modules are simple objects in the category of `R`-modules.
+TODO : prove that reciprocally, a simple object in the category of `R`-modules is indeed
+a simple module.
+-/
+
+section category
+variables {R M : Type*} [ring R] [add_comm_group M] [module R M]
+open category_theory
+open Module
+
+lemma subsingleton_of_epi_zero (N : Module R) [epi (0 : N ⟶ of R M)] : subsingleton M :=
+begin
+  apply subsingleton_of_surjective_zero N,
+  have : function.surjective (0 : N ⟶ of R M) := (Module.epi_iff_surjective _).mp ‹_›,
+  exact this,
+end
```

Comment:
The `‹_›` need some help to find the right hypothesis, but just naming it will work:
```suggestion
lemma subsingleton_of_epi_zero (N : Module R) [h : epi (0 : N ⟶ of R M)] : subsingleton M :=
subsingleton_of_surjective_zero N ((Module.epi_iff_surjective _).mp h)
```